src/main/java/ch/njol/skript/expressions/ExprEntityOwner.java

```diff
@@ -0,0 +1,107 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.UUIDUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.entity.Tameable;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Entity Owner")
+@Description({
+	"The owner of a tameable entity (i.e. horse or wolf) or a dropped item.",
+	"Getting the owner of a dropped item will only return a loaded entity or a player that has played before. "
+	    + "If the entity was killed, or the player has never played before, will return null.",
+	"Setting the owner of a dropped item means only that entity or player can pick it up. "
+		+ "This is UUID based, so it can also be set to a specific UUID.",
+	"Dropping an item does not automatically make the entity or player the owner."
+})
+@Examples({
+	"set owner of target entity to player",
+	"delete owner of target entity",
+	"set {_t} to uuid of tamer of target entity",
+	"",
+	"set the owner of all dropped items to player"
+})
+@Since("2.5, INSERT VERSION (dropped items)")
+public class ExprEntityOwner extends SimplePropertyExpression<Entity, Object> {
+
+	static {
+		Skript.registerExpression(ExprEntityOwner.class, Object.class, ExpressionType.PROPERTY,
+			"[the] (owner|tamer) of %livingentities%",
+			"%livingentities%'[s] (owner|tamer)",
+			"[the] [dropped item] owner of %itementities%",
+			"%itementities%'[s] [dropped item] owner");
```

Comment:
Does this not always match first 2 patterns? if `dropepd item` is not set
```suggestion
			"[the] (owner|tamer) of %livingentities%",
			"%livingentities%'[s] (owner|tamer)",
			"[the] dropped item owner of %itementities%",
			"%itementities%'[s] dropped item owner");
```

Reply:
Ignore this one I see where I miss understood it, but this could probably just be `entities` no?

Reply:
I think it might  be fine as is.